examples/mean-reversion/mean-reversion.go

```diff
@@ -199,12 +199,13 @@ func (alp alpacaClientContainer) rebalance() {
 func (alp alpacaClientContainer) submitLimitOrder(qty int, symbol string, price float64, side string) error {
 	account, _ := alp.client.GetAccount()
 	if qty > 0 {
+		qty := decimal.NewFromFloat(float64(qty))
```

Comment:
[minor] no need for this change

Reply:
I wonder why we cast the int to float64. I believe it can safely be updated to:

```go
decimal.NewFromInt(qty)
```

Reply:
Ironically, `decimal.NewFromInt()` doesn't actually accept `int` arguments: it expects `int64`. Since `rebalance()` declares `qty` to be an `int`, we actually need to cast it to something `decimal` can handle

Reply:
aah fair enough, but wouldn't it still make sense to cast to int64 instead then and not deal with floating points?

Reply:
yeah, you're right. it looks like this implements a discrete mean reversion strategy anyway, so no need to introduce floats here